app/controllers/ConfirmationController.scala

```diff
@@ -56,6 +58,9 @@ class ConfirmationController @Inject()(
         } else {
           Future.successful(Ok(view(data.metaData, data.confirmationViewBreakdown, config.calculatorVersion)))
         }
+      case Valid(data: PhaseTwoConfirmationDataResult) =>
+        Future.successful(Ok(phaseTwoView(data.confirmationViewBreakdown, data.metaData.claimPeriod, config.calculatorVersion)))
```

Comment:
what happens to `auditing` incase of `PhaseTwo` ?

Reply:
will be implemented under a different ticket, CJRSC-253